src/Mod/TechDraw/Gui/QGIViewBalloon.h

```diff
@@ -183,6 +184,7 @@ public Q_SLOTS:
 
     TechDraw::DrawView* getSourceView() const;
     bool m_dragInProgress;
+    bool m_originDragged = false;
```

Comment:
This bool m_originDragged is set false only in the class header, i.e. during object construction time.  As it is set true in balloonLabelDragged() and tested later, shouldn't it also be set false after line 534 where m_dragInProgress is set false?

Reply:
> shouldn't it also be set false after line 534

Yes. I'll fix this later.

Reply:
fixed